src/pages/staking-hub/wrapper.tsx

```
@@ -156,13 +167,15 @@ function WrapperPage() {
   const [wxhoprValue, set_wxhoprValue] = useState('');
   const [showTxInfo, set_showTxInfo] = useState(false);
   const [notEnoughBalance, set_notEnoughBalance] = useState(false);
+  const [handlerIsSet, set_handlerIsSet] = useState<boolean>();
```

ESLint

[error] 'handlerIsSet' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)